src/lib/yang/value.lua

```diff
@@ -95,8 +95,21 @@ function types.identityref.tostring(val)
    return val
 end
 
-types['instance-identifier'] = unimplemented('instance-identifier')
-types.leafref = unimplemented('leafref')
+types['instance-identifier'] = {}
+types['instance-identifier'].parse = function (str, what)
+   return assert(str, 'missing value for '..what)
```

Comment:
SGTM. I will add a comment, same for leafref.

In the midterm I'd like to implement these missing types, but I'd need some guidance because they work a bit different than other supported types and I don't know how to properly implement them (only tried to implement leafref so far). So my plan for the moment if to focus on alarms support.